docs/src/main/tut/reference.md

```diff
@@ -38,11 +38,11 @@ _SQL_: `rf_reproject_geometry`
 Reproject the vector `geom` from `origin_crs` to `destination_crs`. Both `_crs` arguments are either [proj4](https://proj4.org/usage/quickstart.html) strings, [EPSG codes](https://www.epsg-registry.org/) codes or [OGC WKT](https://www.opengeospatial.org/standards/wkt-crs) for coordinate reference systems. 
 
 
-#### rf_extent
```

Comment:
Still need to have `rf_extent` in the docs some place? Or does that require more work in terms of introducing the ProjectedRasterTile concept?